pkgs/development/python-modules/libagent/default.nix

```diff
@@ -16,7 +17,8 @@ buildPythonPackage rec {
     trezor mnemonic ledgerblue
```

Comment:
Is there a reason that these packages aren't in `propagatedBuildInputs`?

Reply:
I hadn't touched that part so I don't know. I checked `setup.py` here: https://github.com/romanz/trezor-agent/blob/master/setup.py#L17
It looks like they should be in `propagatedBuildInputs` and also there are some unnecessary dependencies. I'll fix those.